Packs/QualysFIM/Integrations/QualysFIM/QualysFIM.py

```diff
@@ -28,35 +26,36 @@ class Client(BaseClient):
     """
 
     def __init__(self, base_url: str, verify: bool, proxy: bool, auth: tuple):
-        headers = self.get_token_and_set_headers(base_url, auth)
-        super().__init__(base_url=base_url, verify=verify, proxy=proxy, headers=headers)
+        super().__init__(base_url=base_url, verify=verify, proxy=proxy)
+        self._headers = self.get_token_and_set_headers(auth)
 
-    @staticmethod
-    def get_token_and_set_headers(base_url: str, auth: tuple):
+    def get_token_and_set_headers(self, auth: tuple) -> dict:
         """
         Get JWT token by authentication and set headers.
 
         Args:
-            base_url (str): url for authentication.
             auth (tuple): credentials for authentication.
 
         Returns:
-             headers with token.
+            headers with token.
+
+        Raises:
+            DemistoException if authentication request was not successful.
         """
         try:
-            data = {
-                'username': auth[0],
-                'password': auth[1],
-                'token': True}
+            data = {'username': auth[0], 'password': auth[1], 'token': True}
             headers = {'ContentType': 'application/x-www-form-urlencoded'}
-            token = requests.post(url=f'{base_url}/auth',
-                                  headers=headers,
-                                  data=data).text
+
+            auth_response = self._http_request(method='POST', url_suffix='/auth',
+                                               headers=headers, data=data, resp_type='response')
+            auth_response.raise_for_status()
```

Comment:
Updated method call to `BaseClient._http_request` in commit f42a4199c3dd04121863525acec3441a8c5dc3ec, new / updated kwargs:
* `raise_on_status = True`  _(instead of `auth_response.raise_for_status()`)_
* `resp_type = 'text'`  _(instead of `token = auth_response.text`)_